Add 15.11 Release Notes (#9907)
chef/chef: RELEASE_NOTES.md

This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes/> for the official Chef release notes.

# Chef Infra Client 15.11

> ## Bootstraping Bugfixes

Reviewer: ```suggestion
## Bootstrapping Bugfixes
```

This release of Chef Infra Client resolves multiple issues when using `knife bootstrap` to bootstrap new nodes to a Chef Infra Server:

- Bootstrapping from a Windows host to a Linux host with an ED25519 ssh key no longer fails
- Resolved failures in the Windows bootstrap script
- Incorrect paths when bootstrapping Windows nodes have been resolved

## Chef InSpec 4.18.114

Chef InSpec was updated from 4.18.104 to 4.18.114 with the following improvements:

- Added new `--reporter_message_truncation` and `--reporter_backtrace_inclusion` reporter options to truncate messages and suppress backtraces.

> - Fix for a warning when an input is provided

Reviewer 2: ```suggestion
- Fixed a warning when an input is provided
```

- Inputs and controls can now have the same name

## Resource Improvements

### windows_firewall

> The `windows_firewall` resource has been updated to supports firewall rules that are associated with more than one profile. Thanks [@tecracer-theinen](https://github.com/tecracer-theinen).

Reviewer: ```suggestion
The `windows_firewall` resource has been updated to support firewall rules that are associated with more than one profile. Thanks [@tecracer-theinen](https://github.com/tecracer-theinen).
```

> The `windows_firewall` resource has been updated to supports firewall rules that are associated with more than one profile. Thanks [@tecracer-theinen](https://github.com/tecracer-theinen).

Reviewer 2: ```suggestion
The `windows_firewall` resource has been updated to support firewall rules that are associated with more than one profile. Thanks [@tecracer-theinen](https://github.com/tecracer-theinen).
```
